assets/armsnippets.jsonc

```diff
@@ -78,7 +78,7 @@
         "description": "ARM Template Variable"
     },
     "Parameter": {
-        "prefix": "arm-param-value",
+        "prefix": "arm-param",

```

Comment:
Made this more consistent

Reply:
Consider also updating here?

https://github.com/microsoft/vscode-azurearmtools/blob/master/assets/jsonsnippets.jsonc#L74

Reply:
shouldn't said I fixed it to match the description.

in template:
    "Parameter": {
        "prefix": "**arm-param**",
        "body": [
            "\"${1:parameter1}\": {",
            "   \"type\": \"${2|string,securestring,int,bool,object,secureobject,array|}\",",
            "   \"metadata\": {",
            "        \"description\": \"${3:description}\"",
            "    }",
            "}"
        ],
        "description": "**ARM Template Parameter**"
    },

In params:
    "Parameter Value": {
        "prefix": "**arm-param-value**",
        "body": [
            "\"${1:parameter1}\": {",
            "    \"value\": \"${2:value}\"",
            "}"
        ],
        "description": "**Parameter Value**"
    }